handlers/tribes_test.go

```diff
@@ -216,7 +237,7 @@ func TestDeleteTribe(t *testing.T) {
 			t.Fatal(err)
 		}
 		chiCtx := chi.NewRouteContext()
-		chiCtx.URLParams.Add("uuid", "mockUUID")
```

Comment:
@aliraza556 you did not assert that the tribe is deleted from the DB using the `db.TestDB.GetTribe()` function

Reply:
@elraphty Fixed